processor/examples/README.md

```diff
@@ -0,0 +1,48 @@
+## Getting started with samples ##
+
+## Building the library
+- Clone the repo and cd to the repo directory
+```bash
+git clone https://github.com/Azure/azure-event-hubs-node.git
+cd processor
+```
+- Install typescript, ts-node globally (required for running the samples)
+```bash
+npm i -g typescript
+npm i -g ts-node
+```
+- NPM install from the root of the package
+```bash
+npm i
+```
+- Build the project
+```bash
+npm run build
+```
+
+## Before executing a sample
+- Go to the [Azure Portal](https://portal.azure.com).
+- Here are the docs which would help you create a event hubs and eventhub/iothub resources in the portal :
+  - [Azure Event Hubs - NodeJS DOCS](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-node-get-started-send).
+  - [Azure IoT Hubs - NodeJS DOCS]
```

Comment:
I would add document link to`Azure IoT Hubs - NodeJS DOCS` as well.

Reply:
Oops, I was going to add the link but I forgot. Fixed